documents/manifesto.md

```diff
@@ -0,0 +1,28 @@
+# The Lua Users Foundation Manifesto
+
+The Lua Users Foundation (the Foundation) is an association of individuals in a modular umbrella structure consisted of several workgroups, detailed in the Lua Users Foundation Constitution.
```

Comment:
The constitution document will be drafted in the future.

Reply:
NB: PR comments are the same thing as in-document comments --- as they become much less visible once PR is merged